clickstream/start.sh

```diff
@@ -4,9 +4,9 @@
 . ../utils/helper.sh
 
 check_env || exit 1
-check_running_cp 5.0 || exit 
-check_running_elasticsearch 5.6.5 || exit 1
-check_running_grafana 5.0.3 || exit 1
+check_running_cp 5.0.1 || exit 
```

Comment:
Is there a reason CP *must* be running 5.0.1?  Will it fail with 5.0.0 or 5.0.2?  If it works with 5.0.0 and 5.0.2, then why not leave as 5.0?

Reply:
there's a bug in RUN SCRIPT which was fixed in 5.0.1, so yes it'll fail in 5.0.0. 
The dep is basically >=5.0.1